src/test/ui/parser/let-binop.rs

```diff
@@ -0,0 +1,8 @@
+fn main() {
+    let a: i8 *= 1; //~ ERROR can't reassign to an uninitialized variable
+    let _ = a;
+    let b += 1; //~ ERROR can't reassign to an uninitialized variable
+    let _ = b;
+    let c *= 1; //~ ERROR can't reassign to an uninitialized variable
```

Comment:
What happens if it's already initialized?
```rust
let d = 1;
let d += 2;
```

Reply:
https://play.rust-lang.org/?version=nightly&mode=debug&edition=2018&gist=e06e05de1651f41857588ffad51deaa8:

```
error: can't reassign to an uninitialized variable
 --> src/main.rs:3:7
  |
3 | let d += 2;
  |       ^^ help: initialize the variable
```

Reply:
Yes, but one could argue I just initialized it above (same name).

Reply:
Maybe change the wording for initialized variables? Can't reassign to initialized variable, it should only be either `let d = 2` or `d += 2`, can we even use `let` with `+=`.

Reply:
`+=` cannot be used with `let` to my knowledge. Perhaps we could just say:
```
note: only `=` can be used in `let` bindings
help: change this `+=` for `=`
help: remove `let` (if it was initialized before)
`´`